src/pages/profile-edit.html

```diff
@@ -0,0 +1,75 @@
+          <input type="number" class="w-[230px] h-[50px] bg-[#F7F9FC] text-right pr-3 " value="09152663045">
+        </div>
+        <div class="flex flex-col gap-2 relative ">
+          <label class=" w-[88px] h-[17px]  not-italic font-light text-[12px] leading-[16 .8px] text-[#7E8AAB] mt-5 ">آدرس ایمیل</label>
+          <input type="email" class="  w-[230px] h-[50px] bg-[#F7F9FC] text-right pr-3 " value="efwabifawafup@gmail.com ">
+          <img  class="  absolute bottom-3.5 left-7 ml-20 " src="/src/assets/images/pages/ui/crusore-icon.png" alt="crusore-image">
+        </div>
+      </div>
+  
+      <div class="grid grid-cols-2 gap-x-6">
+        <div class="flex flex-col gap-2 relative">
+          <label class="w-[88px] h-[17px]  not-italic font-light text-[12px] leading-[16 .8px] text-[#7E8AAB] mt-5 mr-5">استان</label>
+          <input type="text" class=" w-[454px] h-[56px] bg-[#F7F9FC] text-right pr-3  mr-5 mt-25 " placeholder="انتخاب کنید">
+          <img  class="  absolute bottom-5 right-20 mr-[350px]" src="/src/assets/images/pages/ui/Arrows-icon.png" alt="crusore-image">
+        </div>
+        <div class="flex flex-col gap-2">
+          <label class="w-[88px] h-[17px]  not-italic font-light text-[12px] leading-[16 .8px] text-[#7E8AAB] mt-5">شهر</label>
+          <input type="text"  class="w-[454px] h-[56px] bg-[#F7F9FC] text-right pr-3 mt-25" placeholder="نام شهر را وارد کنید" >
+        </div>
+        <div class="grid grid-cols gap-x-6 ">
+          <div class="flex flex-col gap-2 items-center relative">
+            <label class="w-[88px] h-[17px]  not-italic font-light text-[12px] leading-[16 .8px] text-[#7E8AAB] mr-5 mt-5">کدپستی</label>
+            <input type="number" class="w-[454px] h-[56px] bg-[#F7F9FC] text-right pr-3  mr-5 mt-25 ml-25 " value="2484789461">
+            <img  class="  absolute bottom-4 right-28 mr-2" src="/src/assets/images/pages/ui/crusore-icon.png" alt="crusore-image">
+          </div>
+          </div>
+          <div class="grid grid-cols gap-x-6 ">
+          <div class="flex flex-col gap-2">
+            <label class="w-[88px] h-[17px]  not-italic font-light text-[12px] leading-[16 .8px] text-[#7E8AAB] mt-5">آدرس</label>
+            <input type="text" class="w-[450-px] h-[56px] bg-[#F7F9FC] text-right pr-3  ml-5 mt-25 " value="ششششششششششششششششصیصش" />
+          </div>
+          </div>
+          <div class="grid grid-cols  ">
+            <div class="flex flex-col items-center relative ">
+              <button type="submit" class="  w-[930px] h-[60px] bg-[#00ADB5] absolute right-7 top-8  ">
+                <span class="w-[102px] h-[25px] mt-[17.5px] font-semibold text-[18px] leading-[25.2px] text-[#FFFFFF] text-center">ذخیره تغییرات</span>
+              </button>
+              <img class="  absolute top-12 right-10" src="/src/assets/images/pages/ui/profile-circle.png"   alt="profile-image">
```

Comment:
Remove any unnecessary spaces between attributes & values.
Use Prettier formater

```suggestion
              <img class="absolute top-12 right-10" src="/src/assets/images/pages/ui/profile-circle.png" alt="profile-image">
```